### ISSUE TITLE
[Suggestion] Overview which project is compatible with RM1/RM2 device

### ISSUE THREAD

**Issue by Stinosko** [reHackable/awesome-reMarkable#91]
Could there be some kind of table which list the compatibility of the projects for Remarkable 1 or 2? So RM2 knows which projects aren't ported yet and if a new project is added to which device it's compatible with. 🙂

**Comment by seifertm** [comment]
Good idea! I don't have a preference how the result should look. I just wanted to point out that the [Oxide](https://github.com/Eeems/oxide) project uses badges for this purpose.

**Comment by Stinosko** [comment]
I was more thinking about something like this:

| Project        | RM1           | RM2  |
| ------------- |:-------------:| -----:|
| App1      | yes | untested |
| App2      | no      |   yes |
| App3 | yes      |    yes |

**Comment by beardhatcode** [comment]
The :one: and :two: emojis could also be used as "badges".

**Comment by faroit** [comment]
maybe 

* project¹ - a project that supports rm1
* project² - a project that supports rm2
* project¹² - a project that supports both

would work as well and less disturbing to the reading flow?

**Comment by Barabazs** [comment]
But who will keep track of the compatibility of each repo? 
A repo could be updated to support the RM2, while it would still be listed as RM1 only in this list.

**Comment by Ameb** [comment]
This will be a feature of toltec, for stuff installed on the device. 

https://github.com/toltec-dev/toltec/issues/129

**Comment by akif5723** [comment]
Hi everybody, I’m maybe in wrong case but maybe someone can help, i look for a hack who allowing the screen side by side like onyx boox note aire or Fujitsu quaderno ??? Please it would be great to have a hack like this somebody know something about a heck like this ???

**Comment by Eeems** [comment]
Ideally, each repo will keep track of their own compatibility and indicate it in their readme (https://remarkable.guide/devel/github.html). Toltec does keep track of all of this as well, for the set of packages it contains.
